app/routes/docs.contributing.github/route.mdx

```diff
@@ -0,0 +1,120 @@
+---
+handle:
+  breadcrumb: GitHub
+---
+
+import copyUrl from './copy-url.png'
+import githubFork from './github-fork.png'
+import githubImage from './github.png'
+import newFork from './new-fork.png'
+
+# GitHub
+
+GCN consists of several open-source projects grouped into the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization. These include:
+
+- [nasa-gcn/gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov): This web site
+- [nasa-gcn/gcn-schema](https://github.com/nasa-gcn/gcn.nasa.gov): The GCN Notices unified schema
+- [nasa-gcn/gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python), [nasa-gcn/gcn-kafka-js](https://github.com/nasa-gcn/gcn-kafka-js): Kafka client libraries
+  ...
+
+To contribute to any project in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, follow these steps:
+
+- [Fork a copy of our repo](#forking-a-repo)
+
+- [Clone your repo to your machine](#cloning-a-repo)
+
+- [Create a branch for your new changes](#branching-off-main)
+
+- [Create a Pull Request to contribute to the original project](#pull-requests)
```

Comment:
- [ ] Avoid unnecessary abbreviations: replace "repo" with "repository."
- [ ] Can the items in this list match the titles of the sections verbatim?